[Bug 701] Fix a misuse of ieee->mode with ieee->iw_mode.

Signed-off-by: Zhu Yi <yi.zhu@intel.com>
---

 ipw2200-1.0.6-chuyee/ipw2200.c |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)

diff -puN ipw2200.c~reset-mode-fix ipw2200.c
--- ipw2200-1.0.6/ipw2200.c~reset-mode-fix	2005-07-15 14:21:25.000000000 +0800
+++ ipw2200-1.0.6-chuyee/ipw2200.c	2005-07-15 14:22:34.000000000 +0800
@@ -8093,7 +8093,7 @@ static int ipw_sw_reset(struct ipw_priv 
 	priv->power_mode = IPW_POWER_AC;
 	priv->tx_power = IPW_TX_POWER_DEFAULT;
 
-	return old_mode == priv->ieee->mode;
+	return old_mode == priv->ieee->iw_mode;
 }
 
 
_
